Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM ubuntu:latest
```

Comment:
we should really replace this by something more lightweight in the future, but it does the job for now